matbench/tests/test_task.py

```diff
@@ -204,7 +205,38 @@ def test_get_info(self):
         self.assertTrue("SHA256 Hash Digest" in mbt.info)
 
     def test_record(self):
+        self._test_record()
+
+    def test_record_std(self):
+        # ensure that conversion from std to ci is correct
+        mbt = MatbenchTask(self.test_datasets[0], autoload=True)
+        _, test_outputs = mbt.get_test_data(0, as_type="tuple", include_target=True)
+        n_test = len(test_outputs)
+        mbt.record(0, [0.0] * n_test, std=[1.0] * n_test)
+        ci_0_0 = list(mbt.results["fold_0"]["uncertainty"].items())[0][1]
+        ci_lower = ci_0_0["ci_lower"]
+        ci_upper = ci_0_0["ci_upper"]
+        # mean:0 std:1, 95% CI-->c.a. 1.96 https://stackoverflow.com/a/29562808/13697228
+        ci_upp_check = 1.959963984540054
+        ci_low_check = -ci_upp_check
+        msg = f"Conversion from normal distribution standard deviation to lower 95% confidence bound ({ci_upper}) is not within tolerance of expected confidence bound ({ci_upp_check})"
+        self.assertAlmostEqual(ci_upper, ci_upp_check, msg=msg)
+        msg = f"Conversion from normal distribution standard deviation to upper 95% confidence bound ({ci_lower}) is not within tolerance of expected confidence bound ({ci_low_check})"
+        self.assertAlmostEqual(ci_lower, ci_low_check, msg=msg)
```

Comment:
Check for simple, known case of std-->ci conversion

Reply:
looks gooe